src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -3290,32 +3324,14 @@ private function matches() {
 	 * Indicates that the parser has found a CDADA node and it's possible
 	 * to read and modify its modifiable text. Note that in HTML there are
 	 * no CDATA nodes outside foreign elements (SVG and MathML). Outside
-	 * of foreign elements, they are treated as HTML comments. Nonetheless,
-	 * the Tag Processor still recognizes them as they appear in the HTML
-	 * stream and exposes them for inspection and modification.
+	 * of foreign elements, they are treated as HTML comments.
```

Comment:
The spec tends to refer to "foreign content":

> CDATA sections can only be used in foreign content (MathML or SVG).

```suggestion
	 * of foreign content, they are treated as HTML comments.
```

Reply:
accepted in 5e017d1bcfb077d42957ad76a631021cb8bc843a

I used elements because I was thinking about being _inside of them_ but you're right, the spec uses both, however, in these cases it's always "in foreign content." thanks 👍